packages/vite/src/node/ssr/ssrModuleLoader.ts

```diff
@@ -296,9 +286,6 @@ async function nodeImport(
       throw err
     }
     url = resolved.id
-    if (usingDynamicImport) {
-      url = pathToFileURL(url).toString()
```

Comment:
I think we need to keep this line. It's failing on Windows.

Reply:
Fixed.